contrib/tabrmd.fc

```diff
@@ -0,0 +1,2 @@
+/usr/sbin/tpm2-abrmd	--	gen_context(system_u:object_r:tabrmd_exec_t,s0)
+/usr/local/sbin/tpm2-abrmd	--	gen_context(system_u:object_r:tabrmd_exec_t,s0)
```

Comment:
Maybe in your commit message you can comment that the policy is for these two paths and if someone uses a different prefix, then the policy has to be modified.

In fact, I think it would be better to have the same information that's in the commit message in the README.